napari/view_function.py

```diff
@@ -48,10 +48,15 @@ def view(
     the returned :class:`napari.Viewer` object.
     """
     viewer = Viewer(title=title)
-    for image in images:
+    if len(*images) > 1:
```

Comment:
I don't think we want to sometimes create a volume layer using the `view` function, this is really just a convenience function for rapidly creating an image layer, and instead people should instantiate a viewer and then call `add_volume` if the want a volume, or - hopefully in not too long we'll be able to support a volume through the image layer. Can you revert this.